usaspending_api/etl/management/commands/load_table_to_delta.py

```diff
@@ -62,7 +62,7 @@
         "is_from_broker": False,
         "source_table": "vw_awards",
         "source_database": "rpt",
-        "destination_database": "raw",
```

Comment:
Is there a reason why we are updating this and not all the other transaction TABLE_SPECs?

Reply:
Good call! I needed to update it for awards, but it wasn't necessary for Transactions. I suspect that making such an update would impact the Transaction Loader tests. I can give it a shot and see what happens.

Reply:
I just changed it back for now until we decide what we want to do with those table_specs.